templates/frontools/themes.json.sample

```diff
@@ -10,8 +10,8 @@
     ]
   },
   "YOUR_THEME_NAME": {
-    "src": "app/design/frontend/Snowdog/YOUR_THEME_NAME",
-    "dest": "pub/static/frontend/Snowdog/YOUR_THEME_NAME",
+    "src": "app/design/frontend/YOUR_VENDOR/YOUR_THEME_NAME",
+    "dest": "pub/static/frontend/snowdog/YOUR_THEME_NAME",
```

Comment:
I think it should be YOUR_VENDOR
from frontools docs:

> dest - full path to pub/static/[theme_area]/[theme_vendor]/[theme_name]

Reply:
Has to stay **Snowdog**